### ISSUE TITLE
Products & Categories gets hidden when i activate the plugin

### ISSUE THREAD

**Issue by BenSegal770** [hyyan/woo-poly-integration#479]
When I make a page in the element that shows products in the language I translate, then it doesn't show them.
And once I turn off the plugin, the products do show up.

**Comment by biguenique** [comment]
Hi, I found your question because I had the very same problem. Turns out I managed to fix this by using the following settings in Languages > Settings > URL Modifications (Polylang settings):

![image](https://user-images.githubusercontent.com/566304/82123456-85c61b80-9767-11ea-814b-c5b9e495f1cc.png)

I can only reproduce the problem when _The language is set from content_ is **selected** and _The front page url contains the language code instead of the page name or page id_ is **unchecked**.

My store page is the main page for the site. Perhaps your situation is different, but I thought this might help someone identify the issue.
